x-pack/legacy/plugins/siem/public/pages/case/components/push_to_service/index.tsx

```diff
@@ -125,25 +108,7 @@ export const usePushToService = ({
       ];
     }
     if (actionLicense != null && !actionLicense.enabledInConfig) {
-      errors = [
-        ...errors,
-        {
-          title: i18n.PUSH_DISABLE_BY_KIBANA_CONFIG_TITLE,
-          description: (
-            <FormattedMessage
-              defaultMessage="The kibana.yml file is configured to only allow specific connectors. To enable opening a case in external systems, add .servicenow to the xpack.actions.enabledActiontypes setting. For more information, see {link}."
-              id="xpack.siem.case.caseView.pushToServiceDisableByConfigDescription"
-              values={{
-                link: (
-                  <EuiLink href="#" target="_blank">
-                    {'coming soon...'}
-                  </EuiLink>
-                ),
-              }}
-            />
-          ),
-        },
-      ];
+      errors = [...errors, getKibanaConfigError()];
```

Comment:
Why don't we use the `getActionLicenseError` helper?

Reply:
same as above we want to keep the order the way it is